exercises/concept/regular-chatbot/.meta/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "blurb": "Learn the basics of Regular Expressions with JavaScript helping a Chatbot understand humans.",
+  "authors": ["julendiaz", "junedev"],
+  "contributors": ["junedev"],
+  "files": {
+    "solution": ["regular-chatbot.js"],
+    "test": ["regular-chatbot.spec.js"],
+    "exemplar": [".meta/exemplar.js"]
+  },
+  "forked_from": ["exercism/javascript"]
```

Comment:
You did not fork the exercise from anywhere so this field should be removed.

Reply:
This is probably because I duplicated a config.json from another exercise (which was possibly forked). 

I'll remove the field.